pkg/parquet/reader.go

```diff
@@ -37,6 +37,7 @@ type (
 type Reader interface {
 	ReadDate(ctx context.Context, datetime time.Time, target interface{}) error
 	ReadDateAsync(ctx context.Context, datetime time.Time, target interface{}, callback ResultCallback) error
+	ReadFileIntoTarget(ctx context.Context, file string, target interface{}) error
```

Comment:
We might also want to be able to define a batch size and an offset if that is possible. That is to not use a GB of memory just because we are deserializing all data of a file at once.